main.rb

```diff
@@ -0,0 +1,71 @@
+require_relative "game_output"
+require_relative "game_input"
+require_relative "turn_master"
+require_relative "game_board"
+require_relative "adjudicator"
+require_relative "ai"
+require_relative "player"
+require_relative "game_modes"
+
+# -----MAIN LOOP-----
+
+while true
+
+    GameOutput.new.start_screen()
+    game_mode =  gets.strip
+    system("cls") || system("clear")
+
+    # Choose Game Mode Loop
+    while true
+        
+        # Player vs Player
+
+        if game_mode == "1"
+            game_board = GameBoard.new
+            game_squares = game_board.squares
+            GameMode.new.p_v_p(game_squares)
+            puts "Press ENTER to continue"
+            gets
+            break
+
+        # Player vs AI
+
+        elsif game_mode == "2"
+            game_board = GameBoard.new
+            game_squares = game_board.squares
+            GameMode.new.p_v_ai(game_squares)
+            puts "Press ENTER to continue"
+            gets
+            break
```

Comment:
we need to get rid of the repeated logic

Reply:
done